.github/workflows/auth-cd.yaml

```diff
@@ -0,0 +1,110 @@
+name: Continuous Deployment - Auth Service
```

Comment:
Naming it auth instead of authentication as it was intended for authentication and authorization. But as of now, there is only authentication checks. Will change to authentication service then.

Reply:
Ah I see, what we gonna use authentication and authorization for ah? What's the difference between them?